.github/workflows/linux.yml

```diff
@@ -94,7 +94,7 @@ jobs:
         CXXFLAGS: ${{matrix.cxxflags}}
       run: |
         cmake -DCMAKE_BUILD_TYPE=${{matrix.build_type}} ${{matrix.fuzz}} ${{matrix.shared}} \
-              -DCMAKE_CXX_STANDARD=${{matrix.std}} -DFMT_DOC=OFF \
+              -DFMT_CXX_STANDARD=${{matrix.std}} -DFMT_DOC=OFF \
```

Comment:
We should use the standard mechanism of specifying standard version.

Reply:
In this case we can just drop the second commit